packages/generator/src/generator.ts

```diff
@@ -229,7 +229,7 @@ export abstract class Generator<
 
   async write(): Promise<void> {
     debug("Generator.write...")
-    const paths = readDirRecursive(this.sourcePath(), (name) => {
+    const paths = await readdirRecursive(this.sourcePath(), (name) => {
       const additionalFilesToIgnore = this.filesToIgnore()
       return ![...alwaysIgnoreFiles, ...additionalFilesToIgnore].includes(name)
     })
```

Comment:
Probably the dotfiles are getting filtered by this callback, right?

Reply:
I've rewritten `readdirRecursive` to check the call to `fs.readdir`, and it doesn't appear. Try adding a `console.log` below `fs.readdir` in `readdir-recursive.ts` and create a new blitz app